common/views/components/WorkHeader/WorkHeader.js

```diff
@@ -61,7 +61,7 @@ const WorkHeader = ({ work, childManifestsCount = 0 }: Props) => {
               [font('hnm', 2)]: true,
               'inline-block': true,
             })}
-            lang={work.language && work.language.id}
+            lang={work.languages.length > 0 && work.languages[0].id}
```

Comment:
Question - do we have a concept of "primary language"? Or what language the catalogued `title` is in?

For instance, [the example you give](https://api.wellcomecollection.org/catalogue/v2/works/m6pm7b7g?include=languages), this would produce an english title with the `lang` code of `chi`.

Reply:
We don't expose one explicitly in the API, at least. (The `"language"` field in the API is just the first entry in `"languages"`.)

* Miro and METS don't include language data
* Calm throws it all into a string field. We parse the languages out as best we can, but it's not especially structured.
* Sierra has different MARC tags for "language" and "additional languages" – we previously used the first tag as the only language we sent, and it will always appear first in the list of new languages

So at best I can say it's no worse than the current status quo?

Reply:
I would say we would rather remove this attribute is we know we are going to populate it incorrectly.

Unless we can talk to the collections info team to try determine what language we are using in the metadata. Ping @gestchild

Reply:
[nod]

Yeah, I couldn't work out where this attribute actually goes. I think it gets passed to the UV, but I couldn't see if it was actually being used.

Reply:
[It's an accessibility attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang) to tell machines what language an element is in.

Reply:
yeah, I think no attribute is better than the wrong value

Reply:
I've had a peek in ES for some stats:

<table>
<tr><th># of languages</th><th># of works</th></tr>
<tr><td>0</td><td>2,286,821</td></tr>
<tr><td>1</td><td>893,609</td></tr>
<tr><td>2 or more</td><td>22,902</td></tr>
</table>

Would it be worth setting the `lang` attribute on the cases where there's only a single language?

i.e. `work.languages.length == 0 && work.languages[0].id`

Reply:
☝️ That's good.